preprocessing.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+import pandas as pd
+from joblib import Parallel, delayed
+from googletrans import Translator
+
+translator = Translator()
+
+features_to_translate = ['category_name', 'city', 'description', 'param_1', 'param_2', 'param_3',
+                         'parent_category_name', 'region', 'title']
+
+
+def translate(filepath, filepath_en):
```

Comment:
@pknut I'd rather have it return csv and then we can simply save it. It is usually a good idea to have function that does one thing.